testnets/xiontestnet/assetlist.json

```diff
@@ -0,0 +1,138 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "xiontestnet",
+  "assets": [
+    {
+      "description": "The native staking token of the Xion network.",
+      "denom_units": [
+        {
+          "denom": "uxion",
+          "exponent": 0,
+          "aliases": [
+            "microxion"
+          ]
+        },
+        {
+          "denom": "mxion",
+          "exponent": 3,
+          "aliases": [
+            "millixion"
+          ]
+        },
+        {
+          "denom": "XION",
+          "exponent": 6,
+          "aliases": [
+            "xion"
+          ]
+        }
+      ],
+      "base": "uxion",
+      "display": "XION",
+      "name": "xion",
+      "symbol": "XION"
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/D934516FBE457F3A98AFABD87E0EFF7F95A15325C191EA8CDD7763C702FDDEC2",
+          "exponent": 0,
+          "aliases": [
+            "uaxl"
+          ]
+        },
+        {
+          "denom": "axl",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/D934516FBE457F3A98AFABD87E0EFF7F95A15325C191EA8CDD7763C702FDDEC2",
+      "name": "Axelar AXL Token",
+      "display": "axl",
+      "symbol": "AXL",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelar",

```

Comment:
sorry, I missed this before. These should be the testnet chains, e.g., 'axelartestnet' not 'axelar'.

Reply:
Nice catch. I fixed all three; `axelartestnet`, `nobletestnet` and `osmosistestnet`